src/VisualStudio/Core/SolutionExplorerShim/AnalyzerItem/AnalyzerItemSource.cs

```diff
@@ -180,5 +180,28 @@ public object SourceItem
                 return _analyzersFolder;
             }
         }
+
+        private static ImmutableArray<AnalyzerItem> GetSortedAnalyzerItems(IEnumerable<AnalyzerItem> analyzerItems, Project project)
+        {
+            // We want to display the analyzers sorted by their display name, but place analyzers with no rules towards the end (analyzer dependencies).
+            var itemsWithRules = new List<AnalyzerItem>();
+            var itemsWithoutRules = new List<AnalyzerItem>();
+            foreach (var item in analyzerItems)
+            {
+                if (item.AnalyzerReference.GetAnalyzers(project.Language).IsDefaultOrEmpty)
```

Comment:
this will create entry in error list if analyzer failed to load due to some issue right? want to make sure that it won't create duplicated entries though.